odl/contrib/fom/supervised.py

```diff
@@ -305,10 +339,11 @@ def range_difference(data, ground_truth, mask=None, normalized=False,
 
     Parameters
     ----------
-    data : `Tensor`
-        Input data to compare to the ground truth.
-    ground_truth : `Tensor`
-        Reference to compare ``data`` to.
+    data : `Tensor` or `array-like`
+        Input data to compare to the ground truth. If not a `Tensor`, a
+        default space with cell size 1 will be assumed.
```

Comment:
cell size 1 is not quite true, we create a simple R^n space, right?

Reply:
True, forgot to update the wording.